testing_text.txt

```diff
@@ -0,0 +1,3 @@
+Hi I´m Alexa but not from Amazon.
```

Comment:
😅

Reply:
@lscantillowl  hahaha now it is my personal jk